backend/tools/keycloak/sync/Models/ClaimModel.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+
+namespace Pims.Tools.Keycloak.Sync.Models
+{
+
```

Comment:
/nit this one is missing a comment.

Reply:
They are inconsistent...  I'm coding like a mad man atm...

Reply:
yeah, fair. Feel free to tech debt it for later, not much point in commenting code that may be heavily modified/removed in the near term.